### ISSUE TITLE
Question about Figure 4 in the Paper

### ISSUE THREAD

**Issue by michaelku1** [er-muyue/DeFRCN#33]
Hello, Thanks again for your contribution in FSOD work.

In the paper, there is a study on the degree of contribution of RPN GDL and RCNN GDL towards the performance, which is summarized in Figure 4. 

<img width="406" alt="Screenshot 2022-01-15 at 17 27 12" src="https://user-images.githubusercontent.com/48415065/149617010-5b599dc6-5a7a-4836-b431-f47cf73c00ce.png">

I would like to know what is the metric for these numbers? Are they AP50 for VOC or mAP for COCO? (are these numbers from VOC or COCO)? Thanks.

**Comment by er-muyue** [comment]
Hi, the Figure 4 summarizes the mAP metric for COCO in the FSOD setting. The (a) is coco base-training and (b) is 10-shot novel fine-tuning.
